src/test/java/teammates/logic/core/NotificationsLogicTest.java

```diff
@@ -55,14 +63,43 @@ private void testGetNotification() throws Exception {
         assertNull(notifLogic.getNotification("nonexistant-notification"));
     }
 
+    private void testGetAllNotifications() throws Exception {
+        ______TS("success: retrieve all when there exist notifications");
+
+        n = getTypicalNotification();
+        notifDb.createEntity(n);
+        NotificationAttributes n1 = getTypicalNotification();
+        n1.setNotificationId("new-notification-id");
+        notifDb.createEntity(n1);
+
+        List<NotificationAttributes> retreivedNotifs = notifLogic.getAllNotifications();
```

Comment:
```suggestion
        List<NotificationAttributes> retrievedNotifs = notifLogic.getAllNotifications();
```
Do make the same modification to any other instances of this typo as well!